client/src/pages/GalleryPage.js

```diff
@@ -1,20 +1,40 @@
 import React from "react";
-import styled from "styled-components";
+import styled from "styled-components/macro";
+import { useEffect } from "react";
+import { getData } from "../utils/api";
+import useAsync from "../utils/useAsync";
 
 const Display = styled.div`
   border: solid 1px lightgray;
 `;
 
-const GalleryPage = () => (
-  <div>
-    <section>
-      <h2>Das hier ist die Album-Seite 🤩</h2>
-    </section>
-    <Display>
-      <img src="" alt="testImage" />
-      <p>Display für Response der GET-Route</p>
-    </Display>
-  </div>
-);
+const GalleryPage = () => {
+  const { data, loading, error, doFetch } = useAsync(() => getData("2"));
```

Comment:
Is this for testing?

Reply:
Yes, the argument "2" is just to see if it works.